thoth/storages/graph/models.py

```diff
@@ -364,13 +361,34 @@ class CVE(Base, BaseExtension):
     python_package_version_entities = relationship("HasVulnerability", back_populates="cve")
 
 
+class BuildLogAnalyzerRun(Base, BaseExtension):
+    """A class representing a single buildlogs-analyzer (build log analysis) run."""
+
+    __tablename__ = "build_log_analyzer_run"
+
+    id = Column(Integer, primary_key=True, autoincrement=True)
+    build_log_analyzer_name = Column(String(256), nullable=True)
+    build_log_analyzer_version = Column(String(256), nullable=True)
+    build_log_analysis_document_id = Column(String(256), nullable=False)
+    datetime = Column(DateTime, nullable=False)
+    debug = Column(Boolean, nullable=False, default=False)
+    build_log_analyzer_error_reason = Column(String(16384), nullable=True)
```

Comment:
Please use `Text` type all over the place. There is no drawback of using it in PostgreSQL.

Reply:
Thanks for the tip ;). Fixed it . PTAL :)